modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator_{context}"]
+= Installing the AWS Load Balancer Operator
+
+You can install the AWS Load Balancer Operator using the Red Hat {product-title} Operator Hub.
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+.Procedure
+
+. Click *Operators* → *OperatorHub* in the OpenShift Web Console.
+. Click *AWS Load Balancer Operator*.
+You can use the Filter by keyword text box or the filter list to search for AWS Load Balancer Operator from the list of operators.
+. Select `aws-load-balancer-operator` namespace.
+. Follow the instructions to prepare the operator for installation.
+. On the AWS Load Balancer Operator page, click *Install*.
+. On the Install Operator page, ensure that you selected the following options:
```

Comment:
If you are referring to a specific page names, like * Install Operator* page, that needs to be marked in asterisks

Reply:
Ack'ed